examples/howto/Model_Builder.myst.md

```diff
@@ -0,0 +1,251 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Using ModelBuilder class for deploying PyMC models 
```

Comment:
You need to add the notebook target right before the header as shown in https://docs.pymc.io/en/latest/contributing/jupyter_style.html#first-cell

Reply:
This file is auto generated after executing `pre-commit run --all`

Reply:
Yes, I add the review comments on the myst one because it is where all this issues can be correctly seen, but you should make the changes to the ipynb file and you will continue to do so in future reviews too. I generally only [comment this](https://github.com/pymc-devs/pymc-examples/pull/419#pullrequestreview-1136572558) in the first review